x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/geo_view/geo_field.tsx

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
For consistency with other components, I think `description.tsx`, `geo_field_select.tsx` and `geo_field.tsx` should be moved out one level into a folder called `geo_field`.
So it is the same as `population_field`, `rare_field`, `split_field` etc

Reply:
Moved in 9be34fed1d464e8fac17090c2aa5e2e78099e150